### ISSUE TITLE
CI does not test julia 1.3 (which is in compat) | QOB.jl tests fail on julia 1.3

### ISSUE THREAD

**Issue by Krastanov** [qojulia/QuantumOpticsBase.jl#139]
I noticed this when reviewing #133 so I decided to install julia 1.3 and run the tests. They fail.

@david-pl , @ChristophHotter , @amilsted , should we try to fix the tests or should we bump the julia compat to 1.6? I personally lean towards bumping the compat given our limited manpower.

**Comment by Krastanov** [comment]
actually, it is a bit worse:

- current master has dependencies that can not install on julia 1.3 as you can see in the CI of #140 
- the latest version that can install on julia 1.3 is 0.3.7 and its tests fail.

The exact errors is:

```
/test/test_operators_lazytensor.jl:381
  Test threw exception
  Expression: all(map(isconcretetype, T.parameters))
  type UnionAll has no field parameters
```

Because of this, I think it is best to bump the compat bound to julia 1.6... But we have test failures on 1.6 too (see #141)

**Comment by Krastanov** [comment]
I fixed the julia 1.6 issue in #141

**Comment by ChristophHotter** [comment]
Bumping the julia compat to 1.6 is fine for me.

**Comment by Krastanov** [comment]
David merged the 1.6 bump so I am closing this issue as fixed by #141
